DependencyInjection/JMSDiExtraExtension.php

```diff
@@ -54,6 +55,12 @@ public function load(array $configs, ContainerBuilder $container)
         $container->setParameter('jms_di_extra.disable_grep', $config['disable_grep']);
         $container->setParameter('jms_di_extra.doctrine_integration', $config['doctrine_integration']);
 
+        if (!empty($config['annotation_patterns']) && is_array($config['annotation_patterns'])) {
+            foreach ($config['annotation_patterns'] as $annotation_pattern) {
+                JMSDiExtraBundle::addAnnotationPattern($annotation_pattern);
```

Comment:
Please use ``$container->setParameter()`` and ``$container->getParameter()`` instead. Third party bundles can change this bundle configuration to do what they want.